src/mapml/layers/TemplatedTileLayer.js

```diff
@@ -62,12 +62,12 @@ export var TemplatedTileLayer = L.TileLayer.extend({
     },
     createTile: function (coords) {
       let tileGroup = document.createElement("DIV"),
-          tileSize = this._map.options.crs.options.crs.tile.bounds.max.x;
+          tileSize = this.getTileSize();
       L.DomUtil.addClass(tileGroup, "mapml-tile-group");
       L.DomUtil.addClass(tileGroup, "leaflet-tile");
       
-      tileGroup.setAttribute("width", `${tileSize}`);
-      tileGroup.setAttribute("height", `${tileSize}`);
+      tileGroup.setAttribute("width", `${tileSize.x}`);
+      tileGroup.setAttribute("height", `${tileSize.y}`);
```

Comment:
Browsers don't do anything with HTML `width` and `height` attributes on `<div>` elements so I think these can be removed.

Reply:
I believe they were added just to let whoever know the tileSize if they ever looked into the shadow dom, since tiles can contain things other than `<img>` with the tileloadstart event.

Reply:
Okay well, the size is already visible in the style attribute on that same `<div>`, e.g. `style="width: 256px; height: 256px;"`. I don't think it's correct to set invalid attributes for developer convenience.